apps/nova/src/components/dashboard/ChatSummaryWrapper.tsx

```diff
@@ -0,0 +1,40 @@
+import { Route } from "@/types/routes";
+import { useFindUserRooms } from "@litespace/headless/chat";
+import { useUser } from "@litespace/headless/context/user";
+import { ChatSummary, type ChatSummaryProps } from "@litespace/luna/Chat";
+import { orUndefined } from "@litespace/sol/utils";
+import { IRoom } from "@litespace/types";
+import dayjs from "dayjs";
+import { useMemo } from "react";
+
+function organizeRooms(
+  list: IRoom.FindUserRoomsApiRecord[] | null
+): ChatSummaryProps["rooms"] {
+  return (
+    list?.map((room) => ({
+      id: room.roomId,
+      url: Route.Chat.concat("?room=", room.roomId.toString()),
+      name: orUndefined(room.otherMember?.name),
+      image: orUndefined(room.otherMember?.image),
+      message: room.latestMessage?.text || "TODO",
+      sentAt: room.latestMessage?.updatedAt || dayjs().toString(), // TODO
+      read: room.unreadMessagesCount === 0,
+    })) || []
+  );
+}
+
+export const ChatSummaryWrapper = () => {
+  const { user } = useUser();
+  const rooms = useFindUserRooms(user?.id, { size: 4 });
+  const organizedRooms = useMemo(() => organizeRooms(rooms.list), [rooms.list]);
+
+  return (
+    <ChatSummary
+      loading={rooms.query.isLoading || rooms.query.isPending}
```

Comment:
They are the same I think. 
```suggestion
      loading={rooms.query.isLoading}
```

Reply:
Ok So, I've researched a bit more about this:
- `isPending`: there's no data and a query hasn't been made yet.
- `isFetching`: the query is being fetched
- `isLoading`: there's no data, and there's a query currently being fetched. Notice that it's a combination of the above.

it's recommended to use `isPending` [here](https://github.com/TanStack/query/discussions/6297#discussioncomment-7467010)